### PR TITLE
rofi: added dmenu option, added homepage

### PR DESCRIPTION
**Pull request by CoffeeVector**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by schneiderl** [comment]
It might just be me but this dmenu option description doesn't really say anything to me. 
Maybe we can rephrase it to make more clear what the command actually does?

**Comment by CoffeeVector** [comment]
- If we pipe a list of items to stdin, the selected item will be printed (or piped into a different process) to stdout:

Would this be more clear?

**Comment by schneiderl** [comment]
reads better imho @CoffeeVector

File: pages/linux/rofi.md [hunk 1]
```diff
@@ -13,3 +14,7 @@
 - Switch between windows:
 
 `rofi -show window`
+
+- If we pipe a list of items to stdin, the selected item will be printed (or piped into a different process) to stdout:
```
Comment (sbrl) [inline pages/linux/rofi.md]: Sorry to be a pain, but I'm not so sure about this. I think it's the `If` here that's bothering me. What about this?

```suggestion
- Pipe a list of items to stdin and print the selected item stdout:
```

I think we can assume that the reader will know that they can send the standard output of one command into another as the standard input.

**Comment by Gorock** [comment]
I looked a bit into the manpages of Dmenu and rofi to see how this option could be worded.

Personally i would word it like dmenu, which is as follows:

>  dmenu is a dynamic menu for X, which reads a list of newline-separated items from stdin.  When the  user  selects an item and presses Return, their choice is printed to stdout and dmenu terminates.  Entering text will
>        narrow the items to those matching the tokens in the input.


tldr:
rofi reads a list of newline-seperated items from stdin. When the user selects an item, their choice is printed to stdout and rofi terminates.

File: pages/linux/rofi.md (hunk 1 above)
Comment (agnivade) [reply pages/linux/rofi.md]: It is not clear to me how the item is selected. The stdin input is just a list of lines.

Comment (CoffeeVector) [reply pages/linux/rofi.md]: Rofi will open a ui thing that'll let the user select one

Comment (agnivade) [reply pages/linux/rofi.md]: Ok, then we should clarify that. Perhaps say "select an item and then print it to console" or something to that effect.

File: pages/linux/rofi.md [hunk 2]
```diff
@@ -13,3 +14,7 @@
 - Switch between windows:
 
 `rofi -show window`
+
+- Pipe a list of items to stdin and print the selected item stdout:
```
Comment (mebeim) [inline pages/linux/rofi.md]:
```suggestion
- Pipe a list of items to stdin and print the selected item to stdout:
```

Typo.

File: pages/linux/rofi.md [hunk 3]
```diff
@@ -13,3 +14,7 @@
...
+
+`echo "{{Something\nOr\nOther}}" | rofi -dmenu`
```
Comment (mebeim) [inline pages/linux/rofi.md]:
```suggestion
`echo -e "{{Coiche1\nChoice2\nChoice3}}" | rofi -dmenu`
```
`echo` needs `-e` to interpret newlines `\n`. Also, I think using `ChoiceN` for the items is clearer.
